tests/e2e/addons/TA_cim_addon/default/eventtypes.conf

```diff
@@ -0,0 +1,11 @@
+##
+## SPDX-FileCopyrightText: 2021 Splunk, Inc. <sales@splunk.com>
+## SPDX-License-Identifier: LicenseRef-Splunk-8-2021
+##
+##
```

Comment:
Please delete those headers as well, I don't think we need them in the test add-ons here.

Reply:
Done